docs-ref-conceptual/manage-cloud-sets-azure-cli.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Managing Cloud Sets with Azure CLI 2.0
+description: Create, log in, and manage different cloud sets with the Azure CLI 2.0.
+keywords: Azure CLI 2.0, Azure, clouds, datacenters
+author: sptramer
+manager: douge
+
+ms.author: sttramer
+ms.date: 06/27/2017
+ms.topic: article
+ms.prod: azure
+ms.technology: azure
+ms.devlang: azurecli
+---
+
+# Managing Multiple Clouds with Azure CLI 2.0
+
+If you have multiple subscriptions associated with Azure, you may have more than one cloud available. Each cloud has its own associated endpoints and capabilities, and is often associated with a particular region that has different data protection standards or requirements.
+
+To effectively work with multiple clouds, you will need to be able to switch between which is currently active, and possibly create new clouds.
+
+## Listing clouds
+
+You may list your available clouds with the [`cloud list`](/cli/azure/cloud#list) command. This will tell you which cloud is currently active, what its current profile is, and can provide information on regional suffixes and host names.

```

Comment:
No need to code block CLI commands when you link them to reference. We probably do this in other places, but about a month ago we agreed that we just want to roll these as regular text links to the ref.